tests/test_spacy_udpipe.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import spacy
+
+from spacy_udpipe import UDPipeModel, download, load
+
+EN = "en"
+RO = "ro"
+SPACY_VERSION = "2.2.4"
+
+
+@pytest.fixture
+def lang() -> str:
+    return EN
+
+
+@pytest.fixture(autouse=True)
+def download_lang(lang: str) -> None:
+    download(lang=lang)
+
+
+def test_serialization(lang: str) -> None:
```

Comment:
I assume this requires picking support?

Reply:
Actually, it worked fine without it (`test_serialization` was just relocated).